portfolio/src/main/java/com/google/sps/servlets/ShawarmaServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Servlet that returns list of shawarma places.
+ */
+@WebServlet("/shawarma")
+public class ShawarmaServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("application/json");
+        try {
+            InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream("shawarma.json");
+            Scanner scanner = new Scanner(inputStream);
+            StringBuffer stringBuffer = new StringBuffer();
+            while (scanner.hasNext()) {
+                stringBuffer.append(scanner.nextLine());
+            }
```

Comment:
Can't you use GSON's `fromJson` to read the shawarma.json file?

Also, the file name is something important and ideally shouldn't be defined in `doGet`. Maybe create a constant for it?

Reply:
Does it mean that I should create ShawarmaPlace object (which represents an element from json file) and do smth like `Gson gson = new Gson();
  Reader reader = Files.newBufferedReader(Paths.get(Thread.currentThread().getContextClassLoader().getResource(SHAWARMA_FILE).toURI()));
  ShawarmaPlace shawarmaPlace = gson.fromJson(reader, ShawarmaPlace.class);
  reader.close();` in the try block?